utils/variables/_shadows.scss

```diff
@@ -15,4 +15,15 @@ $elevation-shadows: (
     "raised": $elevation2,
     "hovering": $elevation3,
     "floating": $elevation5
-);
\ No newline at end of file
+);
+
+$paper-shadow-1: 0 1px 3px rgba($black, 0.12),
```

Comment:
I saw that the shadows which are defined in invision differ with it's name and value from the shadows here. I don't wanted to make a big bang approach and remove the old shadows and therefore I first created the correct variables and in a second step later we will remove the old ones.